static/actions_footer.html

```diff
@@ -0,0 +1,63 @@
+</table>
+
+<br>
+
+<table border=0 align=center>
+
+<tr>
+  <form style='text-align:center' action='add_user' method='GET'>
```

Comment:
I don't speak `<div>` :-(

Made it compilant wrt to the rules you mentioned.

Vertical alignment of the "Add User" and "Add Question" buttons is now off, but that's fine. The footer is still not sticky at the lower edge though. Could you please look into?